README.md

```diff
@@ -133,7 +133,7 @@ composer create-project fourkitchens/sous-drupal-project:dev-[branch-name] PROJE
 ## Contributing without create-project or creating a project with a custom theme
 1. Clone this repository
 2. Checkout the 4.x-beta branch (if it hasn't been released yet)
-3. `composer install` (you must have greater than PHP 7.4 installed locally)
+3. `composer install` (you must have greater than PHP 8.1 installed locally)
```

Comment:
@cbfannin Can we switch this to `lando composer install`?

Reply:
Fixed. Its all your's @birchy82

Reply:
@cbfannin My bad, can we remove the extra comment text too, people don't have to have PHP8.1 installed locally if they are using lando.  I forgot to put that in my previous comment.

Reply:
Fixed.

Reply:
Code looks great, thanks 👍